Add a command line argument `--version` to bin.sh

Yukinobu-Mine opened the pull request:
*Issue #, if available:*
Closes #443

*Description of changes:*
- Add a command line argument `--version` to bin.sh.
- Use the specified version as the tag name for git clone.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Yukinobu-Mine committed:
Add a command line argument `--version` to bin.sh
- Add a command line argument `--version` to bin.sh.
- Use the specified version as the tag name for git clone.

Yukinobu-Mine committed:
Add guidance for super-easy deployment failures

statefb commented on README.md:
````diff
@@ -68,7 +68,7 @@ chmod +x bin.sh
 ```
 
-- You will be asked if a new user or using v1. If so, enter `y`.
+- You will be asked if a new user or using v1. If you are not a continuing user from v0, please enter `y`.
````
曖昧さのケアありがとうございます！！

statefb commented on README.md:
````diff
@@ -93,6 +94,16 @@ You can specify the following parameters during deployment to enhance security a
 ```
 
+> [!TIP]
+> If you see the following output and cannot get the `Frontend URL`, it may be a problem with the latest version. In this case, please add `--version "v1.2.6"` to the parameters and try deployment again.
````
「Frontend URLが出力されない」現象はsuper-easy deployに失敗した場合ですが、仮にデプロイしたとしても、そのほかの何らかのバグにより動作しない可能性があるため、「たとえばfrontend urlが出力されないなど、もしバグがある場合はv1.2.6など過去の特定のバージョンを指定してください」というような書き方の方がより汎用的かなと思いましたが、いかがでしょうか？

statefb commented on README.md:
```diff
@@ -80,11 +80,12 @@ You can specify the following parameters during deployment to enhance security a
 - **--disable-ipv6**: Disable connections over IPv6. (default: enabled)
 - **--allowed-signup-email-domains**: Comma-separated list of allowed email domains for sign-up. (default: no domain restriction)
 - **--bedrock-region**: Define the region where bedrock is available. (default: us-east-1)
+- **--version**: The version of Bedrock Claude Chat to deploy. (default: v1)
```
実際にはバージョン（タグ）とブランチの両方が指定可能だと思いますが、少し紛らわしいかなとも思いました。このパラメータを指定したい場合はminor versionまで含めて具体的に指定すると思いますので、ここでのデフォルト表記は (default: automatically deploy the latest version) などと記載した方がわかりやすいと思いましたが、いかがでしょうか？

**--version**: The version of Bedrock Claude Chat to deploy. (default: automatically deploy the latest version)

Yukinobu-Mine committed:
Reflect review comments for pull request #444
- https://github.com/aws-samples/bedrock-claude-chat/pull/444#discussion_r1678686731
- https://github.com/aws-samples/bedrock-claude-chat/pull/444#discussion_r1678690201

statefb commented:
LGTM!
